bodhi/tests/server/test_models.py

```diff
@@ -867,6 +867,38 @@ def test_all_releases(self):
         assert releases is model.Release.all_releases()
 
 
+class TestReleaseFlatpak(ModelTest):
+    """Unit test case for the ``Release`` model for flatpak releases."""
+    klass = model.Release
+    attrs = dict(
+        name=u"F11F",
+        long_name=u"Fedora 11 Flatpaks",
+        id_prefix=u"FEDORA-FLATPAK",
+        version=u'11',
+        branch=u'f11',
+        dist_tag=u"f29-flatpak",
```

Comment:
Should we change the 11's to 29's?

Reply:
I forgot to adjust each copy of DB entry from 29 to 11. But we can finally leave 29. PR updated :)